Cargo.toml

```diff
@@ -22,6 +22,7 @@ license = "MIT OR Apache-2.0"
 homepage = "https://cot.rs"
 repository = "https://github.com/cot-rs/cot"
 keywords = ["web", "framework", "server", "backend"]
+readme = "README.md"
```

Comment:
```suggestion
readme = "../README.md"
```

Reply:
I thought so, too, but apparently that's not correct. See https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table:

> `license-file` and `readme` are relative to the workspace root

So `"README.md"` is correct.